atd-vze/package.json

```diff
@@ -18,6 +18,7 @@
     "@coreui/icons": "0.3.0",
     "@coreui/react": "^2.5.1",
     "@handsontable/react": "^3.1.3",
+    "@turf/bbox": "^6.5.0",
     "@turf/difference": "6.0.1",
```

Comment:
I wonder if you think it would be worth it to bring `@turf` up to `^6.5.0` so that we don't have two versions of the same library in play? It seems like we might be opening ourselves up to a really nasty bit of debugging if we're passing around GIS data with version changes unbeknownst to the methods we're using.

Reply:
That is a good catch - I didn't see that there was another turf method pulled in here. I'll take a look at this for sure. 🔍🙏

Reply:
@frankhereford On second thought, it doesn't look like this library is used anywhere in the app. I'm going to create a backlog issue because there is some clean up to be done beyond this library, and I'd like to be careful about removing things.

Reply:
That sounds great, and I totally appreciate the idea of not yoinking it out without due care. One less dependency is one less thing dependabot can gripe at us about too. Thanks for looking into it @mddilley!